.helm/ecamp3/templates/api_configmap.yaml

```diff
@@ -19,6 +19,6 @@ data:
   {{- if .Values.php.sentryDsn }}
   SENTRY_API_DSN: {{ .Values.php.sentryDsn | quote }}
   {{- else }}
-  SENTRY_API_DSN: null,
+  SENTRY_API_DSN: {{ "" | quote }}
```

Comment:
An explanation for that would help

Reply:
Sentry doesn't like the dsn property to be null --> fails with an error
Empty string seems to be ok